src/app/3d/qgs3dmaptool.h

```diff
@@ -44,8 +44,8 @@ class Qgs3DMapTool : public QObject
     //! Mouse cursor to be used when the tool is active
     virtual QCursor cursor() const;
 
-    //! Update the connection to the map canvas's property (scene, map setting)
-    virtual void updateSignalSlots();
+    //! Called when canvas's map setting is changed
+    virtual void onMapSettingsChanged();
```

Comment:
Final request, can you make this private? If this is ever moved to public api, it's aleady done.

Reply:
ah yes, it should be private. Update.